renderer/containers/Activity/ActivityActions.js

```diff
@@ -1,17 +1,22 @@
 import { connect } from 'react-redux'
-import { changeFilter, fetchActivityHistory, updateSearchText } from 'reducers/activity'
+import {
+  changeFilter,
+  fetchActivityHistory,
+  updateSearchText,
+  activitySelectors,
+} from 'reducers/activity'
 import ActivityActions from 'components/Activity/ActivityActions'
 
 const mapDispatchToProps = {
   changeFilter,
   fetchActivityHistory,
-  updateActivitySearchQuery: updateSearchText,
+  updateActivitySearchText: updateSearchText,
```

Comment:
Maybe go with updateSearchText to keep it even shorter? this is `ActivityActions.js ` so probably should be obvious anyways what search text is it

Reply:
Done